Orange/widgets/data/owselectrows.py

```diff
@@ -145,6 +142,36 @@ def _plural(s):
     return s
 
 
+class CalendarWidgetWithTime(QCalendarWidget):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.timeedit = QDateTimeEdit(displayFormat="hh:mm:ss")
+        self.timeedit.setTime(self.parent().min_datetime.time())
+
+        self._time_layout = sublay = QHBoxLayout()
+        sublay.setContentsMargins(6, 6, 6, 6)
+        sublay.addStretch(1)
+        sublay.addWidget(QLabel("Time: "))
+        sublay.addWidget(self.timeedit)
+        sublay.addStretch(1)
+        self.layout().addLayout(sublay)
+        self.parent = self.parent()
```

Comment:
Don't store the parent. First, it is accessible as `self.parent()`. Second, you don't need it if you use signals instead (see below).